spec/traits.dd

```diff
@@ -400,6 +401,36 @@ static assert(!__traits(isTemplate,foo!int()));
 static assert(!__traits(isTemplate,"string"));
 ---
 
+$(H2 $(GNAME isZeroInit))
+
+        $(P Takes one argument which must be a type. If the type's
+        $(LINK2 property.html#init, default initializer) has no non-zero bits
```

Comment:
Please don't hard-code HTML links as they won't work in the ebook or PDF.
Use sth. like `$(DDSUBLINK spec/property, init, default initializer)`.
(I now that this macros are poorly documented.)